### PR TITLE
Add UPDATED_SOURCES_REPORTER configuration in JSON schema

### LINKED ISSUE #321: Missing properties in JSON Schema
**Describe the bug**

`.mega-linter.yml` is not validated because of missing *jsonschema* properties.
If found that `UPDATED_SOURCES_*` are missing but maybe some others should be missing too.

```plain
❌ [ERROR] .mega-linter.yml
    Validating .mega-linter.yml against schema from https://raw.githubusercontent.com/nvuillam/mega-linter/master/megalinter/descriptors/schemas/megalinter-configuration.jsonschema.json ...
    
    Errors:
    [
      {
        keyword: 'additionalProperties',
        dataPath: '',
        schemaPath: '#/additionalProperties',
        params: { additionalProperty: 'UPDATED_SOURCES_REPORTER' },
        message: 'should NOT have additional properties'
      }
    ]
    
    ❌ .mega-linter.yml is invalid
```

**To Reproduce**

1. Go to your project working directory
2. Declare `UPDATED_SOURCES_REPORTER: false` in `.mega-linter.yml`
3. Run Mega-Linter (with the YAML V8R)
4. See error in V8R report

**Expected behavior**
`UPDATED_SOURCES_REPORTER` should be accepted as described in the [documentation](https://nvuillam.github.io/mega-linter/reporters/UpdatedSourcesReporter/#configuration)

**Additional context**

Run in a local docker container using:
```bash
docker run  -v $(pwd):/tmp/lint nvuillam/mega-linter:v4
```

**Workaround**

Excludes `.megalinter.yml` from V8R Linter using `YAML_V8R_FILTER_REGEX_EXCLUDE: '.mega-linter.yml'`

### PR DESCRIPTION
**Pull request by JimmyTournemaine**
Fixes #321 

## Proposed Changes

1. Fixing missing UPDATED_SOURCES_REPORTER and UPDATED_SOURCES_REPORTER_DIR from configuration JSON Schema

### Author/Contributor
- [ ] If documentation is needed for this change, has that been included in this pull request

### Reviewing Maintainer
- [ ] Label as `breaking` if this is a large fundamental change
- [ ] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

**Comment by JimmyTournemaine** [comment]
@nvuillam docs/CHANGELOG.md have been edited when I run the `build.sh` script.
I can revert it if needed ;)

**Comment by nvuillam** [comment]
Its is updated only if there are new version of linters ^^
For manual update, please make sure you update the root changelog and not the one in docs folder :)

**Comment by JimmyTournemaine** [comment]
Of course it is, I'm just dump (and I should open the file cause it seems that I don't know hot to read a git diff) :)

File: megalinter/descriptors/schemas/megalinter-configuration.jsonschema.json [hunk 1]
```diff
@@ -6820,6 +6820,16 @@
       "title": "TYPESCRIPT_STANDARD: Including Regex",
       "type": "string"
     },
+    "UPDATED_SOURCES_REPORTER": {
+      "$id": "#/properties/UPDATED_SOURCES_REPORTER",
+      "title": "TYPESCRIPT_STANDARD: Activates/deactivates reporter",
```
Comment (nvuillam) [inline megalinter/descriptors/schemas/megalinter-configuration.jsonschema.json:6825]: Wrong title ^^

File: megalinter/descriptors/schemas/megalinter-configuration.jsonschema.json [hunk 2]
```diff
@@ -6820,6 +6820,16 @@
...
+      "type": "boolean"
+    },
+    "UPDATED_SOURCES_REPORTER_DIR": {
+      "$id": "#/properties/UPDATED_SOURCES_REPORTER_DIR",
+      "title": "TYPESCRIPT_STANDARD: Sub-folder of reports folder containing updated sources",
```
Comment (nvuillam) [inline megalinter/descriptors/schemas/megalinter-configuration.jsonschema.json:6830]: Wrong title ^^

File: megalinter/descriptors/schemas/megalinter-configuration.jsonschema.json (hunk 1 above)
Comment (nvuillam) [reply megalinter/descriptors/schemas/megalinter-configuration.jsonschema.json:6825]: plz set "Activates/Deactivated updated sources reporter"

File: megalinter/descriptors/schemas/megalinter-configuration.jsonschema.json (hunk 2 above)
Comment (nvuillam) [reply megalinter/descriptors/schemas/megalinter-configuration.jsonschema.json:6830]: juste remove the "TYPESCRIPT_STANDARD" :p

**Review by nvuillam — CHANGES_REQUESTED** [review]
Plz make updates requested in comments :)
About CI failing, it's not your fault, i'm working to solve that on master branch so your PR will pass once you merge master into your branch :)

**Comment by JimmyTournemaine** [comment]
so many mistakes... i have squashed my commits to hide my shame and taken into account your comments :)

File: megalinter/descriptors/schemas/megalinter-configuration.jsonschema.json [hunk 3]
```diff
@@ -6820,6 +6820,16 @@
       "title": "TYPESCRIPT_STANDARD: Including Regex",
       "type": "string"
     },
+    "UPDATED_SOURCES_REPORTER": {
+      "$id": "#/properties/UPDATED_SOURCES_REPORTER",
+      "title": "Activates/Deactivated updated sources reporter",
```
Comment (nvuillam) [inline megalinter/descriptors/schemas/megalinter-configuration.jsonschema.json:6825]: "Activates/Deactivates", plz ^^

**Review by nvuillam — CHANGES_REQUESTED** [review]
plz check the comment about the typo ^^

+ Please update CHANGELOG.md
then run `bash build.sh` :)

(cf [Contributing guide](https://nvuillam.github.io/mega-linter/contributing/))

**Comment by nvuillam** [comment]
@JimmyTournemaine t'avais ptetre un peu trop forcé sur le chouchen, ou alors t'étais pas encore totalement redescendu depuis la Rave Party de Lieuron, ça arrive ;)

CI issues are solved in master, please merge master in your branch, fix the typo, update the changelog,  and it will all pass, make you a part of the Mega-Linter contributors Hall Of Fame 😃

**Review by nvuillam — APPROVED** [review]

**Comment by nvuillam** [comment]
Thanks for your contribution ;)

Portsall forever !

**Pull request merged**